sdk/easm/Azure.Analytics.Defender.Easm/stress/README.md

```diff
@@ -0,0 +1 @@
+For information on reliability testing with the Azure SDK stress test framework, please refer to the [Reliability testing wiki](https://aka.ms/azsdk/stress).
```

Comment:
If you're not creating stress tests, let's remove this folder as it serves no purpose.